ui/raidboss/data/03-hw/alliance/weeping_city.js

```diff
@@ -67,13 +68,23 @@
       // only when that boss is in progress.
       id: 'Weeping City HeadMarker Arachne',
       netRegex: NetRegexes.message({ line: '.*Queen\'s Room will be sealed off.*?', capture: false }),
+      netRegexDe: NetRegexes.message({ line: '.*Spinnenfalle will be sealed off.*?', capture: false }),
+      netRegexFr: NetRegexes.message({ line: '.*Domaine de la Tisseuse will be sealed off.*?', capture: false }),
+      netRegexJa: NetRegexes.message({ line: '.*蜘蛛女の狩場 will be sealed off.*?', capture: false }),
+      netRegexCn: NetRegexes.message({ line: '.*女王蛛猎场 will be sealed off.*?', capture: false }),
+      netRegexKo: NetRegexes.message({ line: '.*거미 여왕의 사냥터 will be sealed off.*?', capture: false }),
       run: function(data) {
         data.arachneStarted = true;
       },
     },
     {
       id: 'Weeping City HeadMarker Ozma',
       netRegex: NetRegexes.message({ line: '.*Gloriole will be sealed off.*?', capture: false }),
+      netRegexDe: NetRegexes.message({ line: '.*金字塔上层 will be sealed off.*?', capture: false }),
+      netRegexFr: NetRegexes.message({ line: '.*Aureole will be sealed off.*?', capture: false }),
```

Comment:
I'm surprised this does not give extra errors given that the regex replacement is `The Gloriole`.  Would you mind removing `.*` from the beginning of each of these zone regexes and changing `Queen's Room` and `Gloriole` to `The Queen's Room` and `The Gloriole`? Sorry for not catching this the first time around.

Reply:
done